src/components/Main.js

```diff
@@ -18,8 +18,8 @@ export default class Main extends React.Component {
     ) : (
       ""
     );
-    let details = isBrowser ? (
-      <details>
+    let keyboardControls = isBrowser ? (
+      <details className="keyboard-controls">
```

Comment:
If I'm not wrong, we already use `isBrowser` from [react-device-detect](https://www.npmjs.com/package/react-device-detect) for conditional rendering. So we don't need to use CSS for this, I think.